raiden/tests/smart_contracts/test_decoder_netting_channel.py

```diff
@@ -149,3 +152,94 @@ def test_decode_refund_transfer(
     refund_transfer.sign(PrivateKey(privatekey0, ctx=GLOBAL_CTX, raw=True), address0)
 
     assert_decoder_results(refund_transfer, decoder)
+
+
+def test_decode_tampered_direct_transfer(tester_state, tester_nettingchannel_library_address):
+    privatekey0 = tester.DEFAULT_KEY
+    address0 = privatekey_to_address(privatekey0)
+
+    decoder = deploy_decoder_tester(tester_state, tester_nettingchannel_library_address)
+
+    direct_transfer = make_direct_transfer()
+    direct_transfer.sign(PrivateKey(privatekey0, ctx=GLOBAL_CTX, raw=True), address0)
+
+    message_encoded = direct_transfer.encode()
+    transfer_raw, _ = decoder.getTransferRawAddress(message_encoded)
+
+    names_slices = compute_slices(messages.DirectTransfer.fields_spec)
+    for name, slice_ in names_slices.iteritems():
+        if name == 'signature':
+            continue
+
+        tampered_transfer = bytearray(transfer_raw)
```

Comment:
yeah I guess you are right. The code is self-explanatory. Sometimes I like a good well-placed comment in tests to make the reading easier but perhaps here it's unnecessary.